src/plugins/home/server/services/sample_data/data_sets/ecommerce/saved_objects.ts

```diff
@@ -91,7 +91,6 @@ export const getSavedObjects = (): SavedObject[] => [
         defaultMessage: '[eCommerce] Markdown',
       }),
       visState:
-        // '{"title":"[eCommerce] Markdown","type":"markdown","params":{"fontSize":12,"openLinksInNewTab":false,"markdown":"### Sample eCommerce Data\\nThis dashboard contains sample data for you to play with. You can view it, search it, and interact with the visualizations. For more information about OpenSearch Dashboards, check our [docs](https://www.opensearch.org/guide/en/kibana/current/index.html)."},"aggs":[]}',
         '{"title":"[eCommerce] Markdown","type":"markdown","params":{"fontSize":12,"openLinksInNewTab":false,"markdown":"### Sample eCommerce Data\\nThis dashboard contains sample data for you to play with. You can view it, search it, and interact with the visualizations. For more information about OpenSearch Dashboards, check our [docs](https://github.com/opensearch-project)."},"aggs":[]}',
```

Comment:
Should the `docs` links in these files route to https://opensearch.org/docs/latest/dashboards/index/ instead?